### ISSUE TITLE
 Duplicate response

### ISSUE THREAD

**Issue by Ryan2018-js** [scalar/scalar#2500]
### What happens?
Duplicate response in  @scalar/nestjs-api-reference.  vsrsion is 0.3.107
![Snipaste_2024-07-16_15-40-29](https://github.com/user-attachments/assets/f22e6e9e-9f31-44c8-b757-8962cf72e70c)

### What did you expect to happen?

Response appears only once

### How can we reproduce the issue?

schema content like this.
{
  "schema": {
    "allOf": [
      {
        "type": "object",
        "properties": {
          "data": {
            "type": "object"
          },
          "code": {
            "type": "number"
          },
          "message": {
            "type": "string"
          }
        },
        "required": [
          "data",
          "code",
          "message"
        ]
      },
      {
        "type": "object",
        "properties": {
          "data": {
            "type": "object",
            "properties": {
              "list": {
                "type": "array",
                "items": {
                  "type": "object",
                  "properties": {
                    "id": {
                      "type": "number"
                    },
                    "userName": {
                      "type": "string"
                    },
                    "nickName": {
                      "type": "string"
                    },
                    "email": {
                      "type": "string"
                    },
                    "show": {
                      "type": "boolean"
                    },
                    "phoneNumber": {
                      "type": "string"
                    },
                    "createTime": {
                      "format": "date-time",
                      "type": "string"
                    },
                    "updateTime": {
                      "format": "date-time",
                      "type": "string"
                    },
                    "roles": {
                      "type": "array",
                      "items": {
                        "type": "number"
                      }
                    }
                  },
                  "required": [
                    "id",
                    "userName",
                    "nickName",
                    "email",
                    "show",
                    "phoneNumber",
                    "createTime",
                    "updateTime",
                    "roles"
                  ]
                }
              }
            }
          }
        }
      }
    ]
  }
}

When I switch to the  Swagger UI in NestJS, there are no issues, and there are no duplicates displayed
![Snipaste_2024-07-16_14-55-53](https://github.com/user-attachments/assets/eeffbabe-47f3-4b00-83c2-afc1f59018d1)

**Comment by hanspagel** [comment]
Thanks for the report! The good news is, that’s fixed in #2451 already, which was released in #2434 as `@scalar/api-reference@1.24.40`. The NestJS plugin is using the latest version from the CDN, so it should include the fix already.

Long story short. Would you be able to hit the refresh button really hard and confirm it’s still there? Best way is to open the Chrome developer tools, right click on the reload button and select “Hard Reload”:

<img width="300" alt="Screenshot 2024-07-16 at 10 16 47" src="https://github.com/user-attachments/assets/5cb0d6e8-b94e-446f-bebc-39c5c3a54953">

I hope that fixes the bug already, but I’ll keep it open until we can really confirm it’s gone.

**Comment by Ryan2018-js** [comment]
> Thanks for the report! The good news is, that’s fixed in #2451 already, which was released in #2434 as `@scalar/api-reference@1.24.40`. The NestJS plugin is using the latest version from the CDN, so it should include the fix already.
> 
> Long story short. Would you be able to hit the refresh button really hard and confirm it’s still there? Best way is to open the Chrome developer tools, right click on the reload button and select “Hard Reload”:
> 
> <img alt="Screenshot 2024-07-16 at 10 16 47" width="300" src="https://private-user-images.githubusercontent.com/1577992/349000238-5cb0d6e8-b94e-446f-bebc-39c5c3a54953.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExMTk1ODYsIm5iZiI6MTcyMTExOTI4NiwicGF0aCI6Ii8xNTc3OTkyLzM0OTAwMDIzOC01Y2IwZDZlOC1iOTRlLTQ0NmYtYmViYy0zOWM1YzNhNTQ5NTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTZUMDg0MTI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MThjZGVkYzFkMjkxZDU3YmE0OWRmYjI2ZDdiNGE5MTAwZjMyMWNiNzExZjM3NTQ2NmU1NjY0ZmIyMTNjYzBiZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.WX_XaoF0JLzGW4sbJCLQiy4mhjuX5JPFh6yjAnyoL3w">
> I hope that fixes the bug already, but I’ll keep it open until we can really confirm it’s gone.

Thank you for your response. After refreshing, it has returned to normal.
I noticed another issue. When I copy the OpenAPI to https://sandbox.scalar.com, everything works fine. However, when I copy it to https://docs.scalar.com, the response is still repeating, even after refreshing.

**Comment by hanspagel** [comment]
Oh, yes! You're right. We’re slowly rolling out new versions, https://docs.scalar.com is mostly the last platform we update. It’s our goal to keep that one as stable as possible. But the update should be rolling out there in the next days for sure.
